py/src/braintrust/logger.py

```diff
@@ -601,105 +686,48 @@ def __init__(
-        Braintrust will use the `input` to know whether two test cases are the same between experiments, so they should
-        not contain experiment-specific state. A simple rule of thumb is that if you run the same experiment twice, the
-        `input` should be identical.
-        :param output: The output of your application, including post-processing (an arbitrary, JSON serializable object),
-        that allows you to determine whether the result is correct or not. For example, in an app that generates SQL queries,
-        the `output` should be the _result_ of the SQL query generated by the model, not the query itself, because there may
-        be multiple valid queries that answer a single question.
-        :param expected: The ground truth value (an arbitrary, JSON serializable object) that you'd compare to `output` to
-        determine if your `output` value is correct or not. Braintrust currently does not compare `output` to `expected` for
-        you, since there are so many different ways to do that correctly. Instead, these values are just used to help you
-        navigate your experiments while digging into analyses. However, we may later use these values to re-score outputs or
-        fine-tune your models.
-        :param scores: A dictionary of numeric values (between 0 and 1) to log. The scores should give you a variety of signals
-        that help you determine how accurate the outputs are compared to what you expect and diagnose failures. For example, a
-        summarization app might have one score that tells you how accurate the summary is, and another that measures the word similarity
-        between the generated and grouth truth summary. The word similarity score could help you determine whether the summarization was
-        covering similar concepts or not. You can use these scores to help you sort, filter, and compare experiments.
-        :param metadata: (Optional) a dictionary with additional data about the test example, model outputs, or just
-        about anything else that's relevant, that you can use to help find and analyze examples later. For example, you could log the
-        `prompt`, example's `id`, or anything else that would be useful to slice/dice later. The values in `metadata` can be any
-        JSON-serializable type, but its keys must be strings.
-        :param id: (Optional) a unique identifier for the event. If you don't provide one, Braintrust will generate one for you.
-        :param dataset_record_id: (Optional) the id of the dataset record that this event is associated with. This field is required if and only if the
-        experiment is associated with a dataset.
-        :param inputs: (Deprecated) the same as `input` (will be removed in a future version)
+        Log a single event to the experiment. The event will be batched and
+        uploaded behind the scenes.
+
+        :param **event: Data to be logged. See `braintrust.log` for full details.
```

Comment:
maybe switch the documentation, i.e. keep it here and delete in `braintrust.log()`? I think the latter is rarely (if ever) used